src/components/NFTPage/PriceSection.vue

```diff
@@ -103,6 +105,11 @@ export default {
       default: false,
     },
   },
+  data() {
+    return {
+      hasHoverSellButton: false,
```

Comment:
Better set this in page where we send the actual log, instead of component?
Hover event should always emit, but we only want to log once